include/swift/Frontend/Frontend.h

```diff
@@ -578,6 +581,11 @@ class CompilerInstance {
                                  OptionSet<TypeCheckingFlags> TypeCheckOptions);
 
   void finishTypeChecking(OptionSet<TypeCheckingFlags> TypeCheckOptions);
+
+public:
+  PrimarySpecificPaths getPSPsForWMO() const;
```

Comment:
WMO *might* be an exception to the acronyms rule because most of the team already knows it, but even that is iffy, especially without a comment.

Reply:
Spelled it out.